### DISCUSSION TITLE
Add auto-blocking parameters to adaptive covariance methods

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1583] [Ideas/Science]
Reference [here](https://arxiv.org/pdf/1503.05621.pdf). Think this approach or others like it (I haven't searched thoroughly) may considerably speed up sampling for many ODE models.

**Comment by MichaelClerx** [comment]
> We propose an automated procedure to determine an efficient MCMC algorithm for a given model and computing platform. Our procedure dynamically determines blocks of parameters for joint
sampling that result in efficient sampling of the entire model.
